src/data/terraform.json

```diff
@@ -153,7 +153,7 @@
 		},
 		{
 			"iconName": "docs",
-			"name": "Plugin",
+			"name": "Plugin Development",
```

Comment:
Have updated this one property to make it clear how we're intending to match design. I have a task to update remaining `rootDocsPaths` icons and names, intent is to handle that separately: 🎟️  [[terraform] update docs sub paths icons and text](https://app.asana.com/0/1202097197789424/1202691941299957/f)